utils/api-scripts/src/helpers/extrinsics.ts

```diff
@@ -40,17 +41,122 @@ export class ExtrinsicsHelper {
     return nonce
   }
 
+  async sendAndCheckSudo(
+    tx: SubmittableExtrinsic<'promise'>,
+    errorMsg?: string,
+    wrap = true
+  ): Promise<ISubmittableResult> {
+    const sudoUri = process.env.SUDO_URI || '//Alice'
+    const sudoMultiKeys = (process.env.SUDO_MULTISIG_KEYS || '').split(',').filter((v) => v)
+    const sudoMultiUris = (process.env.SUDO_MULTISIG_URIS || '').split(',').filter((v) => v)
+    tx = wrap ? this.api.tx.sudo.sudo(tx) : tx
+
+    if (sudoMultiKeys.length && sudoMultiUris.length) {
+      const sudoSigners = sudoMultiUris.map((u) => getKeyFromSuri(u))
+      console.log(`Using multisig sudo`)
+      console.log('Multisig keys:', sudoMultiKeys)
+      console.log('Multisig threshold', sudoSigners.length)
+      console.log(
+        'Signers:',
+        sudoSigners.map((s) => s.address)
+      )
+      return this.sendAndCheckMulisig(sudoSigners, sudoMultiKeys, tx, errorMsg)
+    } else {
+      const sudoKey = getKeyFromSuri(sudoUri)
+      console.log(`Using sudo key: ${sudoKey.address}`)
+      return (await this.sendAndCheck(sudoKey, [tx], errorMsg))[0]
+    }
+  }
+
+  async sendAndCheckMulisig(
+    signers: KeyringPair[],
+    keys: string[],
+    tx: SubmittableExtrinsic<'promise'>,
+    errorMsg?: string
+  ): Promise<ISubmittableResult> {
+    if (!signers.length || !keys.length) {
+      throw new Error('sendAndCheckMulisig: signers.length and keys.length must be > 0')
+    }
+    const otherKeys = (s: KeyringPair) =>
+      sortAddresses(
```

Comment:
I was going to ask why sorting is needed then I saw the pallet implementation which checks `ensure_sorted_and_insert`. And now I realize the sorting is to ensure the multi-address derived from the accounts depend on the order :)